stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
I think it would be great if every directory could be built as a standalone cmake project. It seems like that would fit with the cmake philosophy and someone could just go to any swift tool/library and invoke cmake and ninja to build just that. The main thing preventing this design is that everything is invoked through build-script and presets (which can be replaced by cmake caches).